model/models/llama/model.go

```diff
@@ -33,6 +34,13 @@ type Model struct {
 }
 
 func New(c fs.Config) (model.Model, error) {
+	if c.String("tokenizer.ggml.model") == "llama" {
+		return nil, fmt.Errorf("unsupported tokenizer: llama")
```

Comment:
Can you add a comment on why this is unsupported since it's not obvious from the code. It's confusing to see a model identifying itself as llama being rejected